docker/puppetserver/README.md

```diff
@@ -50,6 +50,10 @@ The following environment variables are supported:
 If you would like to do additional initialization, add a directory called `/docker-custom-entrypoint.d/` and fill it with `.sh` scripts.
 These scripts will be executed at the end of the entrypoint script, before the service is ran.
 
+## Persistance 
+
+If you plan to use the in-server CA, restarting the container can cause the server's keys and certificates to change, causing agents and the server to stop trusting each other. To prevent this, you can persist the default ssldir, `/etc/puppetlabs/puppet/ssl`. For example, `docker run -v $PWD/ca-ssl:/etc/puppetlabs/puppet/ssl puppetlabs/puppetserver:latest`.
```

Comment:
```suggestion
If you plan to use the in-server CA, restarting the container can cause the server's keys and certificates to change, causing agents and the server to stop trusting each other. To prevent this, you can persist the default cadir, `/etc/puppetlabs/puppetserver/ca`. For example, `docker run -v $PWD/ca-ssl:/etc/puppetlabs/puppetserver/ca puppetlabs/puppetserver:latest`.
```